Added Appstate module support

munezbn opened the pull request:
Added Appstate turbo module support
Added platform plugin support for ubuntu with dummy app notifications. 
Added memory notification based on cgroup

munezbn committed:
Added Appstate module support

Added platform plugin support for ubuntu with dummy app notification.
Added memory notification based on cgroup

munezbn committed:
Updated plugin library path to /usr/lib

chatrapathi06 commented on ReactSkia/pluginfactory/interfaces/RNSApplicationManagerInterface.h:
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 1994-2021 OpenTV, Inc. and Nagravision S.A.
```
license year should be 2022

munezbn committed:
Addressed review comments

uint64_t is long unsigned in ubuntu and long long unsiggned in TBC. So used long long unsigned
update license year

munezbn committed:
Addressed more review comments

Just specify library name instead of full path. It should load library from default LD_LIBRARY_PATH
